helm-chart/kuberay-operator/templates/ray_raycluster_role.yaml

```diff
@@ -0,0 +1,38 @@
+# permissions for end users to view rayjobs.
+{{- if and .Values.rbacEnable (not .Values.singleNamespaceInstall) }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: raycluster-viewer-role
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+rules:
+- apiGroups:
+  - ray.io
+  resources:
+  - rayjobs
+  - rayjobs/status
```

Comment:
only operator can update/patch status subresource. so viewer can get/list/watch xx/status, while editor not